cmd/kubeadm/app/phases/bootstraptoken/node/tlsbootstrap.go

```diff
@@ -88,3 +90,27 @@ func AutoApproveNodeBootstrapTokens(client clientset.Interface, k8sVersion *vers
 		},
 	})
 }
+
+// AutoApproveNodeCertificateRotation creates RBAC rules in a way that makes Node certificate rotation CSR auto-approved by the csrapprover controller
+func AutoApproveNodeCertificateRotation(client clientset.Interface) error {
+
+	fmt.Println("[bootstraptoken] Configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node certificate rotation")
```

Comment:
Configured RBAC rules to allow certificate rotation for all node client certificates in the cluster